### ISSUE TITLE
Label edges of trajectory

### ISSUE THREAD

**Issue by kverstae** [dynverse/dynplot#31]
Labeling of the edges could help make the trajectory more interpretable.
By adding the edge labels to the dimred plot and the heatmap, it would become easier to link information of both plots to each other, especially in more complex trees.

**Comment by zouter** [comment]
Good point, we'll add this once we work further on the labelling part of the trajectory :slightly_smiling_face:

**Comment by alexQiSong** [comment]
Hi, has this feature been added to the package? I would really look forward to having it. Thanks.
